addons/stock_dropshipping/models/stock.py

```diff
@@ -47,3 +46,25 @@ def _compute_warehouse_id(self):
         super()._compute_warehouse_id()
         if self.default_location_src_id.usage == 'supplier' and self.default_location_dest_id.usage == 'customer':
             self.warehouse_id = False
+
+
+class StockLot(models.Model):
+    _inherit = 'stock.lot'
+
+    def _compute_delivery_ids(self):
+        super()._compute_delivery_ids()
+        for lot in self:
+            if lot.delivery_count > 0:
+                last_delivery = lot.delivery_ids.sorted(key=lambda d: d.date_done, reverse=True)[0]
```

Comment:
Don't think this is a performance bottleneck, but I would use `max()` instead

Reply:
Ya its better I agree